charts/splunk-connect-for-snmp/values.yaml

```diff
@@ -455,8 +537,64 @@ mongodb:
     storageClass: "microk8s-hostpath"
   volumePermissions:
     enabled: true
+  livenessProbe:
+    # The liveness probe checks if the MongoDB instance is running and responsive.
+    exec:
+      # The exec command for the liveness probe to run in the container.
+      command:
+        - mongo
+        - --eval
+        - 'db.adminCommand("ping")'
+    # Number of seconds after the container has started before liveness probes are initiated.
+    initialDelaySeconds: 30
+    # How often (in seconds) to perform the probe.
+    periodSeconds: 10
```

Comment:
Have you checked/read how it affects the performance of MongoDB?

Reply:
Yes you are complete right. I just placed livenessProbe: enabled flag there to decapsulated it from underlaying layers :)